client/my-sites/patterns/components/section/style.scss

```diff
@@ -50,14 +58,3 @@
 .patterns-section__body {
 	@include patterns-page-width;
 }
-
-
-@media (max-width: $break-wide) {
-	.patterns-section {
-		padding: 40px 0 48px;
-	}
-
-	.patterns-section__header {
-		padding-bottom: 32px;
-	}
-}
```

Comment:
Regarding this refactoring, I will try to explain my point of view and then we will decided which approach we will use everywhere.

I always try to separate and encapsulate code, that's why I like approach when we have the next structure:



```
.main-style-1 {...}
.main-style-2 {...}
.main-style-3 {...}

@media (max-width: 1000) { ... }

@media (max-width: 600) { ... }

@media (max-width: 300) { ... }
```



In the example above it's easy to:
1) Read styles from top to bottom
2) Easy to understand overrides, again from top to bottom
3) Also easy to work with breakpoints separately. For example - if I need to make some change, initially I am working with root scope, then I decrease size of my browser to 1000 and work only with `@media (max-width: 1000) { ... }`, then decrease to 600, working there, etc. In this case, I have consistency in development, and as a result in the code, if I need to make changes only for `600` - I will work only in this scope and 100% won't break styles for `> 600`. And as a result, we have less cohesion, and declaration of CSS styles is easy to maintain.

Reply:
There's no documented preference for nesting media queries or not (see [Calypso coding guidelines](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md)). I feel like the choice comes down to a combination of personal preference and a case-to-case basis. 

I originally tweaked the media queries in this file to also use `xlarge` instead of `wide`. However, after some testing, I determined that keeping `wide` was more appropriate. Because I prefer nesting selectors, I didn't revert that particular change, though. All that to say, I didn't make this change just because – it was more a case of "now that I've done it, let's leave it in there".

The reason I like nesting is mostly because I think it makes the CSS easier to read. I'll look up selectors first and expect to see the responsive rules right there. Having to scroll down to a separate media query introduces a risk of forgetting which CSS rules the media query overrides.

We seem to mostly nest media queries in `client/my-sites/patterns` (though there are still a few root-level media queries). Consistency seems good here, which is why I'd argue that this change makes sense. It's not super important to me, though.